cmd/child.go

```diff
@@ -81,14 +81,10 @@ func child(_ *cobra.Command, args []string) error {
 	}
 
 	fmt.Printf("Running command: %v\n", cmdArgs)
-	execCmd := exec.Command(cmdArgs[0], cmdArgs[1:]...)
-	execCmd.Stdin = os.Stdin
-	execCmd.Stdout = os.Stdout
-	execCmd.Stderr = os.Stderr
-
-	if err := execCmd.Run(); err != nil {
+	if err := syscall.Exec(cmdArgs[0], cmdArgs, os.Environ()); err != nil {
 		return fmt.Errorf("error: %v", err)
```

Comment:
(always) use `%w` for errors to wrap them this will give you the ability to later check the error stack for individual errors with `errors.As`
Also add context if you use `fmt.Errorf`, but if you think it unnecessary/are lazy then just `return err`

Reply:
you are printing `error: %v, err` but that does not help. The err object already tells you that it is an error. What you want to do here is add context specific information e.g. `fmt.Errorf("Command %s with args %v failed with error: %w`", cmdArgs[0], cmdArgs, err)